examples/create_letters_from_csv/letter.py

```diff
@@ -0,0 +1,148 @@
+# usage:
+#   python letter.py input.csv
+#   logs to csv's in the output directory
+
+import csv
+import datetime
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../..'))
+
+###########################################################
+# TODO: Provide the following constants
+# Your API Key, keep this secure
+lob.api_key = 'test_fc26575412e92e22a926bc96c857f375f8b'
+###########################################################
+
+# Input check
+if len(sys.argv) < 2:
+    print("Please provide an input CSV file as an argument.")
+    print("usage: python letter.py <CSV_FILE>")
+    sys.exit(1)
+
+# Open input files
+input_filename = sys.argv[1]
+
+success_csv_fields = [
+    'name',
+    'id',
+    'url',
+    'address_line1',
+    'address_line2',
+    'address_city',
+    'address_state',
+    'address_zip'
+]
+errors_csv_fields = ['error']
+
+# create the output directory,
+output_dir = os.path.join('.',  'output')
+if not os.path.isdir(output_dir):
+    os.mkdir(output_dir)
+
+timestamp = datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+
+try:
+    output_dir = os.path.join(output_dir, timestamp)
+    os.mkdir(output_dir)
+except Exception, e:
+    print('Error: %s' % str(e))
```

Comment:
Meh, we could for this one, I learned about the % operator and went HAM with it because I was really excited about it.